chains/evm/cli/deploy/deploy.go

```diff
@@ -61,23 +61,23 @@ var (
 )
 
 func BindDeployEVMFlags(cmd *cobra.Command) {
-	cmd.Flags().BoolVar(&Bridge, "bridge", false, "deploy bridge")
-	cmd.Flags().StringVar(&BridgeAddress, "bridgeAddress", "", "bridge contract address. Should be provided if handlers are deployed separately")
-	cmd.Flags().BoolVar(&DeployAll, "all", false, "deploy all")
-	cmd.Flags().Uint8Var(&DomainId, "domainId", 1, "domain ID for the instance")
-	cmd.Flags().BoolVar(&Erc20, "erc20", false, "deploy ERC20")
-	cmd.Flags().BoolVar(&Erc20Handler, "erc20Handler", false, "deploy ERC20 handler")
-	cmd.Flags().StringVar(&Erc20Name, "erc20Name", "", "ERC20 contract name")
-	cmd.Flags().StringVar(&Erc20Symbol, "erc20Symbol", "", "ERC20 contract symbol")
-	cmd.Flags().BoolVar(&Erc721, "erc721", false, "deploy ERC721")
-	cmd.Flags().BoolVar(&Erc721Handler, "erc721Handler", false, "deploy ERC721 handler")
-	cmd.Flags().StringVar(&Erc721Name, "erc721Name", "", "ERC721 contract name")
-	cmd.Flags().StringVar(&Erc721Symbol, "erc721Symbol", "", "ERC721 contract symbol")
-	cmd.Flags().StringVar(&Erc721BaseURI, "erc721BaseURI", "", "ERC721 base URI")
-	cmd.Flags().BoolVar(&GenericHandler, "genericHandler", false, "deploy generic handler")
-	cmd.Flags().Uint64Var(&Fee, "fee", 0, "fee to be taken when making a deposit (in ETH, decimas are allowed)")
-	cmd.Flags().StringSliceVar(&Relayers, "relayers", []string{}, "list of initial relayers")
-	cmd.Flags().Uint64Var(&RelayerThreshold, "relayerThreshold", 1, "number of votes required for a proposal to pass")
+	cmd.Flags().BoolVar(&Bridge, "bridge", false, "Deploy bridge")
+	cmd.Flags().StringVar(&BridgeAddress, "bridge-address", "", "Bridge contract address. Should be provided if handlers are deployed separately")
+	cmd.Flags().BoolVar(&DeployAll, "all", false, "Deploy all")
+	cmd.Flags().Uint8Var(&DomainId, "domain", 1, "Domain ID for the instance")
```

Comment:
Somewhere domain is Uint64 and somewhere it is Uint8.
In general we should allow domainID to be more than 255 (uint8)

Reply:
Should probably check if there are some inconsistencies with other types.

Reply:
Actually, scratch uint64 it is uint8 on contract so it can't be anything more than that.